lib/parse/blueprint.js

```diff
@@ -81,6 +94,28 @@ module.exports = function(filePath, autoOptions, routeMap) {
                     parseAction(parsedUrl, autoOptionsAction, routeMap);
                 });
             }
+
+            function getBasePath(result, basePathKey) {
+                return result.ast.metadata.reduce(function (basePath, item) {
+                    if (item.name.toLowerCase() === basePathKey.toLowerCase()) {
+                        return '/' + item.value.replace(/^\/|\/$/g, '');
```

Comment:
Can we pull out this regex into a named value? We should have more insight as to what this regex is doing by reading the code.

Reply:
Updated it. I put the regex there in case someone wanted to add a proper path, e.g., /path/to/service/. Not really a problem for our purposes, but figured I'd put a check here.